src/AutoRest.CSharp/Common/Output/Builders/ClientBuilder.cs

```diff
@@ -18,9 +18,11 @@ namespace AutoRest.CSharp.Common.Output.Builders
     internal static class ClientBuilder
     {
         private const string ClientSuffixValue = "Client";
+        private const string RestClientPrefix = "Rest";
```

Comment:
Do we need this one?

Reply:
We only use it in a single place - I'll inline it.